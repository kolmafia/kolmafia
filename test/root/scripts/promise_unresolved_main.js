module.exports = {
  main: () => new Promise(() => {
    // This promise is never resolved.
  })
};
