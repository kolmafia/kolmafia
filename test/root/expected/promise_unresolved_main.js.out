JavaScript exception: Promise did not resolve or reject